components/eamxx/tests/single-process/ml_correction/CMakeLists.txt

```diff
@@ -12,7 +12,13 @@ else()
     message(FATAL_ERROR "pybind11 is missing. Use CMake >= 3.11 or download it")
 endif()
 
+message(STATUS "Python_EXECUTABLE: ${Python_EXECUTABLE}")
```

Comment:
Same as before, these were just for debugging CMAKE vs Pybind found python libraries and could be omitted.